web/router.ex

```diff
@@ -1,6 +1,10 @@
 defmodule RemoteRetro.Router do
   use RemoteRetro.Web, :router
 
+  if Mix.env == :dev do
+    forward "/sent_emails", Bamboo.EmailPreviewPlug
```

Comment:
hmmm... what does this do exactly? i don't see this path referenced anywhere else? my guess is its boilerplate for Bamboo? Is the path prescriptive?